Sources/MapboxCoreNavigation/EHorizon/RoadObjectStore.swift

```diff
@@ -37,7 +37,7 @@ public final class RoadObjectsStore {
 
     /**
      Returns road object with given identifier, if such object cannot be found returns null.
-     NB: since road objects can be removed/added in background we should always check return value for null,
+     NB: since road object can be removed/added in background we should always check return value for null,
```

Comment:
Nit: It’s more grammatical to say either “road objects” or “a road object” when referring to an instance of the RoadObject class in a sentence. (×5)

Reply:
Thanks for the suggestion, I'll improve this part.